src/util/expr.cpp

```diff
@@ -21,13 +21,6 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include <stack>
 
-/// Return whether the expression is a constant.
-/// \return True if is a constant, false otherwise
-bool exprt::is_constant() const
```

Comment:
No, I haven't actually done measurements.

Reply:
Do you think it would be more in-keeping with the PR review checklist you added to either leave the function where it is or to do the performance measurement and include it in the commit message? As a small aside I would have preferred to have seen the performance related change in a separate commit, in order to make it easier to evaluate in isolation.

Reply:
There's a bit of a problem here: replacing `id() == ID_constant` by `.is_constant()` (without inlining it) will entail a function call. So there would inevitably be additional cost, although that need not translate into practically notable cost.

What I could do is not introduce new uses of `is_constant()` and just benchmark the performance impact of inlining.